src/common/rnfs.js

```diff
@@ -0,0 +1,109 @@
+import { Platform } from "react-native";
+import {
+  readDir,
+  DocumentDirectoryPath,
+  MainBundlePath,
+  readFile,
+  writeFile,
+  exists,
+  copyFile,
+  copyFileAssets,
+} from "react-native-fs";
+import { constant, logError } from "@common";
+
+// Paths and URLs
+const BUNDLED_DB_PATH = `${MainBundlePath}/www/${constant.DB}.db`;
+const BUNDLED_MD5_PATH = `${MainBundlePath}/www/${constant.DB}.md5`;
+export const LOCAL_DB_PATH = `${DocumentDirectoryPath}/${constant.DB}.db`;
+const LOCAL_MD5_PATH = `${DocumentDirectoryPath}/${constant.DB}.md5`;
+const REMOTE_BASE_URL = "https://banidb.blob.core.windows.net/database";
```

Comment:
Can this url be moved to somewhere in config or .env variables. Does this project have a .env? If so, this URL can be part of that, it would be easier to change in future if its not hardcoded.